src/main/java/com/github/maleksandrowicz93/cqrsdemo/student/StudentQueryFacade.java

```diff
@@ -0,0 +1,25 @@
+package com.github.maleksandrowicz93.cqrsdemo.student;
+
+import com.github.maleksandrowicz93.cqrsdemo.student.dto.StudentDto;
+import com.github.maleksandrowicz93.cqrsdemo.student.dto.StudentIdentification;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@FieldDefaults(makeFinal = true)
+@RequiredArgsConstructor
+public class StudentQueryFacade {
```

Comment:
Nie jestem przekonany co do konieczności istnienia tej fasady - tylko deleguje wywołania metod `studentQueryHandler`, wydaje się, że spłaszczenie tego będzie korzystne, szczególnie że w przypadku query logika nie powinna się pojawić, więc zawsze będzie to tylko delegacja.